package.json

```diff
@@ -4,7 +4,7 @@
   "description": "An open source and touch friendly web based comic book reader as a react component",
   "main": "./dist/villain.js",
   "scripts": {
-    "start": "webpack-dev-server --config ./webpack.config.js --mode development --open --hot",
+    "start": "webpack-dev-server --config ./webpack.config.js --mode development --open --hot  --port 3000",
```

Comment:
I'm not sure about this change, is this really necessary ?

Reply:
this change was for me because default port 8080 was used by another process

Reply:
Oh, ok, maybe you could open an issue about this to improve the development workflow, thanks :v: ^^